deploy/examples/csi/rbd/storageclass-ec.yaml

```
@@ -59,6 +59,16 @@ parameters:
   # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
   # unmapOptions: force
 
+   # (optional) Instruct the plugin it has to encrypt the volume
+   # By default it is disabled. Valid values are "true" or "false".
+   # A string is expected here, i.e. "true", not true.
+   # encrypted: "true"
+
+   # (optional) Use external key management system for encryption passphrases by
+   # specifying a unique ID matching KMS ConfigMap. The ID is only used for
+   # correlation to configmap entry.
+   # encryptionKMSID: <kms-config-id>
```

I am a little confused by aspects of this, and I think end users might be as well. Perhaps something like below is better. "unique" ID implies to me something like a UUID. Instead, I think what is meant is that it should be "the" ID that corresponds to the KMS configuration in the `rook-ceph-csi-kms-config` configmap's `config.json` data entry.

```suggestion
   # (optional) Use external key management system (KMS) for encryption keys by
   # specifying a unique ID matching a KMS ConfigMap. The ID is only used for
   # correlation to configmap entry.
   # encryptionKMSID: <kms-config-id>
```